lingetic-nextjs-frontend/app/components/questions/FillInTheBlanks/Question.tsx

```
@@ -1,98 +1,47 @@
-            {isChecking ? "Checking..." : "Check"}
-          </button>
-        </div>
-      )}
-      {isError && <p>An error occurred! Please try again after some time.</p>}
-    </>
+    <div>
+      <span>{textBefore}</span>
+      <input
+        type="text"
+        value={value}
+        autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)